examples/helm-charts/cubejs/values.yaml

```diff
@@ -33,6 +33,10 @@ config:
   ##
   apiPort: 4000
 
+  ## The port for a Cube.js deployment to listen to SQL connections on
+  ##
+  sqlPort:
```

Comment:
Do you think it's useful to add the default SQL port here? Similar to how you added the default API port.
I'm just asking. If you think it is not needed, let's skip it.

Reply:
According to the doc: https://cube.dev/docs/backend/sql, the sql API is enabled only if we set a port.

I reflected here this behavior: no value disabled, value enabled. The API is disabled by default.